### ISSUE TITLE
Responsive layout?

### ISSUE THREAD

**Issue by skord** [columbusrb/conclave#87]
Is this a project goal? We had it and it seems to have disappeared - no longer can see the account options on a phone.

**Comment by mdarby** [comment]
I'm back and forth on it -- and I could be totally off base:

I tend to think that responsive-design is a bit over-hyped. iPhone/iPad dominates the market and both resize sites intelligently.

**Comment by skord** [comment]
I'm thinking that I'd love to sit down and listen to an expert talk about bootstrap. Because our layout is quite broken and I have some questions. Try it on an iPhone or the iPhone sim. I'm not sure how to go about fixing it, when fluid and non-fluid is appropriate, etc.

**Comment by mdarby** [comment]
You and me both!

**Comment by skord** [comment]
Let's get a designer in on this and hold them responsible. 

![Old man Cigar](http://www.gentlemenaffair.com/wp-content/uploads/2012/02/old-man-and-cigar.gif)

pic related. what we'll be doing when we can point fingers.

**Comment by mdarby** [comment]
You add a few of these puppies, and I am down:
![asdf](http://unbiasedwriter.com/wp-content/uploads/2011/11/Bulleit-Rye-Manhattan-1.jpg)

**Comment by skord** [comment]
I think I have this sorted out. In my mind at least. I figured it out when I went to use some FontAwesome icons and lo and behold, they weren't there. 

This project started out using LESS and FontAwesome with responsive support (responsive is packaged seperately), but somewhere along the line it disappeared and we moved to using pre-built bootstrap (in vendor). This new setup didn't include the FontAwesome support or the optional responsive support. 

So on my local copy I attempted to switch to a managed bootstrap (bootstrap-sass) and while most things work, many do not (tooltip support for example). It _should_ work, but it's doesn't... thanks asset pipeline. 

So if you ever used a Mac in the pre-OS X days, you might remember troubleshooting system extensions. It's something like that. Remove some of them in different combinations to figure out what works and what doesn't. 

I'm going to stab directly at the heart of the immediate problems which aren't CSS related, but JS related and try to figure out the layout stuff later (which gets really hosed by the responsive addition). 

So yeah, working on this, but if anyone is a genius here I could probably use a hand.
